tests/components/zha/test_light.py

```diff
@@ -67,9 +67,7 @@
 }
 
 
-@asynctest.mock.patch(
-    "zigpy.zcl.clusters.general.OnOff.read_attributes", new=MagicMock()
-)
+@patch("zigpy.zcl.clusters.general.OnOff.read_attributes", new=MagicMock())
```

Comment:
@MartinHjelmare is this what you meant in the other PR?

Reply:
Yes, thanks. :+1: